docs/source/app-dev/grpc/error-codes.rst

```diff
@@ -34,15 +34,15 @@ The goal is to enable users, developers and operators to act on the encountered
 errors in a self-service manner, either in an automated-way or manually.
 
 Configuration
--------------
+======================================
```

Comment:
These is now in accordance with /docs/REAMDE.md](https://github.com/digital-asset/daml/blob/main/docs/README.md#style-conventions)